examples/tfengine/generated/team/bootstrap/main.tf

```diff
@@ -43,6 +43,12 @@ module "project" {
   skip_gcloud_download    = true
   activate_apis = [
     "cloudbuild.googleapis.com",
+    "container.googleapis.com",
```

Comment:
remove these APIs from the CICD module?

Reply:
nvm, you are just adding the additional used service APIs here.

Reply:
Right, that way it's consistent experience when using CICD and not using CICD to enable APIs on the devops project.